docs/src/modules/spring/pages/timers.adoc

```diff
@@ -1 +1,82 @@
 = Timers
+
+include::ROOT:partial$timers-intro.adoc[]
+
+To demonstrate its functionality, let's consider an Ordering Service composed of a xref:value-entity.adoc[Value Entity] and an xref:actions.adoc[Action]. The Action will work as a xref:actions-as-controller.adoc[Controller] receiving calls and delegating to the Order Entity.
+
+Users can place an order, but the order must be confirmed within a period of time. You can compare it with an Ordering Food application where the restaurant needs to confirm if it can accept the order. If no confirmation is sent within some pre-defined period of time, the order is automatically cancelled.
+
+== Order Entity
+Let's have a look on how the Order Entity can be implemented.
+
+.src/main/java/com/example/domain/OrderEntity.java
+[source,java]
+----
+include::example$spring-reliable-timers/src/main/java/com/example/domain/OrderEntity.java[tag=order]
+----
+<1> The first method to look at is the `placeOrder`. It's basically the creation of an order. Note that we set the `placed` field to true.
```

Comment:
I would divide this point in two. One pointing to line 28 on OrderEntity with `The first method to look at is the `placeOrder`. It's basically the creation of an order` and the other in line 33 on OrderEntity with `Note that we set the "placed" field to true` maybe further elaborating why setting `placed` to true is important to note?

Reply:
Also good to change it in java/scala docs.

Reply:
changed now in both. tks